### DISCUSSION TITLE
Font manager for LXQt

### DISCUSSION THREAD

**Discussion by iamthesenate1** [lxqt/lxqt#2037] [Ideas]
I think it would be nice if there was a font manager for LXQt, especially because there is no currently mantained qt font manager gui (all I could find was [fontmatrix](https://github.com/fontmatrix/fontmatrix) which isn't mantained anymore.)

**Comment by tsujan** [comment]
Font manager is like media player, web browser, instant messenger, etc. LXQt has no plan to be a software suite; on the contrary, it's designed so that all DE-agnostic apps can be used with it, whether they're written in Qt or not.

Apps that LXQt provides (like file manager, archive manger, image viewer,...) are either too important for a DE or somehow interrelated. LXQt may have also rewritten or adopted a few apps.

**Reply by iamthesenate1** [reply]
I understand.

On a sidenote it's kinda sad to see that there is no Qt font manager.
